### ISSUE TITLE
HTML / CSS changes

### ISSUE THREAD

**Issue by amjad** [grafana/grafana#15269]
I'm using the latest version of grafana as of 2/5/19 and running it on nginx Ubuntu 18.04 on Digital Ocean. I needed to customize the css a bit directly in the code and I have done that. But the changes are not being reflected on the dashboard. Even though I cleared the cache and restarted the server. 

What do I need to do after making some css or html template changes so that it reflects on the live site?

**Comment by amjad** [comment]
Thanks, when you say update build folder files, is that in addition to making changes to html files?

does that command also work on Ubuntu?

**Comment by amjad** [comment]
@bergquist How do you do this one a live server? I installed it from apt install.

**Comment by torkelo** [comment]
You can’t easily do that, you can modify the optimized and minified css files but that is not easy

**Comment by amjad** [comment]
@bergquist Is there no other way to re-generate or re-build the css and js files in /build directory? I mistakenly deleted some of them!

**Comment by amjad** [comment]
@torkelo Is there no other way to re-generate or re-build the css and js files in /build directory? I mistakenly deleted some of them!

**Comment by torkelo** [comment]
Of course, reinstall grafana

**Comment by amjad** [comment]
@torkelo doesnt that defeat the purpose of customizing grafana html and css? reinstalling will erase the changes right.

**Comment by torkelo** [comment]
Yes, but you said you lost the files , right? Or you download source , make source change and create optimized build from source following are build instructions in docs or readme

**Comment by amjad** [comment]
@torkelo, thanks, ok re-installing brings that those files, I have done that. But then the documentation on your website isn’t clear on how to re-build the front end after customizing the source.

I have installed grafana on the server using `apt-get install grafana` following this tutorial here: https://www.digitalocean.com/community/tutorials/how-to-install-and-secure-grafana-on-ubuntu-16-04

I am looking for clear steps on how to re-build front end after making css or html changes to the code. I’m not sure if that’s too difficult to explain?

**Comment by amjad** [comment]
I see some commands in the readme to automatically rebuild frontend on changes, but there is no instruction about where to run, what folder to run it, what the pre-requisites should be, etc.

**Comment by torkelo** [comment]
you need to start by cloning the GitHub grafana project into a local folder. You can or rebuild for install files

**Comment by amjad** [comment]
@torkelo How do I re-install the same version of grafana (thats already installed) on Ubuntu 18.04? Basically trying to refresh the install so the build files are created again.

**Comment by sergiodegrazia** [comment]
Grafana automatically creates two CSS files at each build or update, one for the 'light' and one for the 'dark' template. The easiest option is to amend these files with the required changes. All changes should last until the next manual update of the application.

In Ubuntu, the files are located at:
/usr/share/grafana/public/build
And are called something like:
grafana.dark.0d52b7637e43802b8d00.css
grafana.light.0d52b7637e43802b8d00.css

I understand this is not a permanent fix, but it's quick an easy to implement, and the results are immediately visible after clearing cache on the browser.

![image](https://user-images.githubusercontent.com/50920877/73080260-73c12400-3ebd-11ea-93f7-2f27d75e1738.png)
